### DISCUSSION TITLE
virtual Table 虚拟表格，在flex布局时并且父容器 overflow: hidden时，会偶现横向足够显示还是会出现横向滚动条

### DISCUSSION THREAD

**Discussion by xiaobing0723** [ant-design/ant-design#51650] [Q&A]
官网示例 https://stackblitz.com/edit/react-ychcvz?file=demo.tsx
![微信图片_20241115154347](https://github.com/user-attachments/assets/cff1a039-8b68-4803-a4e2-459b875c9408)

**Comment by petercat-assistant** [comment]
您提到在使用 Ant Design 的虚拟表格（Virtual Table）时，遇到了在 flex 布局下，父容器设置为 `overflow: hidden` 时，偶尔会出现横向滚动条的问题，即使横向空间足够显示内容。

这个问题可能与以下因素有关：

1. **表格宽度计算**：虚拟表格在渲染时可能会因为宽度计算不准确而导致滚动条出现。确保表格的列宽设置正确，并且总宽度不超过父容器的宽度。

2. **浏览器渲染问题**：不同浏览器对 CSS 的渲染可能会有差异，尤其是在使用 flex 布局时。可以尝试在不同浏览器中测试，看看问题是否一致。

3. **样式冲突**：检查是否有其他 CSS 样式影响了表格的布局，尤其是涉及到宽度和溢出的样式。

4. **Ant Design 版本问题**：确保使用的是最新版本的 Ant Design，可能在新版本中已经修复了相关问题。

如果以上方法无法解决问题，建议在 Ant Design 的 GitHub 仓库中查看是否有类似的 issue，或者提交一个新的 issue 以获得社区的帮助。

如需进一步协助，请在评论中描述您的问题并 @petercat-assistant 以开始与我对话。
